packages/network-controller/src/NetworkController.test.ts

```diff
@@ -1608,13 +857,13 @@ describe('NetworkController', () => {
               createMetamaskProviderMock.mockReturnValue(fakeMetamaskProvider);
 
               await waitForStateChanges(messenger, {
-                propertyPath: ['isCustomNetwork'],
+                propertyPath: ['currentNetworkType'],
                 after: () => {
                   controller.setProviderType(networkType);
                 },
               });
 
-              expect(controller.state.isCustomNetwork).toBe(false);
+              expect(controller.state.currentNetworkType).not.toBe('custom');
```

Comment:
Instead of testing that it's not "custom", should we test that it's "infura"? This would match the test name.